Tests/iaas/mandatory-services/mandatory-iaas-services.py

```diff
@@ -235,7 +234,7 @@ def main():
         format="%(levelname)s: %(message)s",
         level=logging.DEBUG if args.debug else logging.INFO,
     )
-    openstack.enable_logging(debug=args.debug)
+    openstack.enable_logging(debug=False)
```

Comment:
Why?
If we enable debug with `--debug`, we should honor it, no?

Reply:
Comparing the outputs: Without the change, we get some debugging from both openstacksdk and boto(s3), with it only from s3.

Reply:
Yes, makes sense.